FunctorCategories/examples/TerminalCategory.g

```diff
@@ -1,25 +1,30 @@
 #! @Chunk TerminalCategory
 
 #! @Example
-LoadPackage( "FunctorCategories", false );
+LoadPackage( "FunctorCategories" );
+#! true
+LoadPackage( "Locales" );
```

Comment:
```suggestion
LoadPackage( "FunctorCategories", false );
#! true
LoadPackage( "Locales", false );
```
Otherwise the generated `tst` file cannot be executed standalone without errors.

Reply:
Thanks, done.

Btw, how can one execute the test file with gap without it complaining about differences in whitespace?

Reply:
> Btw, how can one execute the test file with gap without it complaining about differences in whitespace?

Either you have to hack https://github.com/gap-system/gap/blob/577dcecf9bff8ba8357546983785d77d658b76f3/lib/init.g#L953 to include the option `compareFunction` (just as in `tst/testall.g`), our you have to make sure the whitespace is correct in your example files (by removing `compareFunction` in `tst/testall.g` and replacing the expected strings by the found strings). The latter is the correct solution but is time-consuming. A faster way would be to remove the example file, keep the generated tst file (using a new name, e.g. the name of the original example file), and set the option `rewriteToFile` in `tst/testall.g` once.